hnn_core/viz.py

```diff
@@ -1069,3 +1069,44 @@ def _onclick(event):
 
     plt_show(show)
     return ax.get_figure()
+
+
+
+def csd_generate(csd, times, ax=None, colorbar=True, show=True):
```

Comment:
```suggestion
def plot_csd(csd, times, ax=None, colorbar=True, show=True):
```

all the plotting functions start with `plot_`

Reply:
also the function needs to be added in `api.rst`